### PR TITLE
Add enableDebug blurb

### PR DESCRIPTION
**Pull request by markwolff**
Addresses #750 
Unsure of if it should be placed in a "Debug Settings" section, or if just having a paragraph in the config table is fine.

### CODE REVIEW

File: AISKU/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 [![Build Status](https://travis-ci.org/Microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/Microsoft/ApplicationInsights-JS)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-web.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-web)
 [![minified size size](https://img.badgesize.io/https://1dsjssdk.blob.core.windows.net/scripts/ai.1.min.js.svg?label=minified%20size)](https://img.badgesize.io/https://1dsjssdk.blob.core.windows.net/scripts/ai.1.min.js.svg?label=minified%20size)
-[![gzip size](https://img.badgesize.io/https://1dsjssdk.blob.core.windows.net/scripts/ai.1.min.js.svg?compression=gzip&softmax=24000&max=25000)](https://img.badgesize.io/https://1dsjssdk.blob.core.windows.net/scripts/ai.1.min.js.svg?compression=gzip&softmax=24000&max=25000)
+[![gzip size](https://img.badgesize.io/https://1dsjssdk.blob.core.windows.net/scripts/ai.1.min.js.svg?compression=gzip&softmax=27000&max=30000)](https://img.badgesize.io/https://1dsjssdk.blob.core.windows.net/scripts/ai.1.min.js.svg?compression=gzip&softmax=27000&max=30000)
```
Comment (jpiyali) [inline AISKU/README.md]: do u manually update this? what exactly is the svg file?

File: AISKU/README.md [hunk 2]
```diff
@@ -126,11 +126,11 @@ Most configuration fields are named such that they can be defaulted to falsey. A
...
-| enableDebug | false | If true, internal debugging data is thrown as an exception by the logger. Default is false. |
 | disableExceptionTracking | false | If true, exceptions are no autocollected. Default is false. |
 | disableTelemetry | false | If true, telemetry is not collected or sent. Default is false. |
-| consoleLoggingLevel | 0 | (internal) Logs internal Application Insights errors to console. 0: off, 1: Critical errors only, 2: Everything (errors & warnings) |
-| telemetryLoggingLevel | 1 | (internal) Sends internal Application Insights errors as telemetry. 0: off, 1: Critical errors only, 2: Everything (errors & warnings) |
+| enableDebug | false | If true, **internal** debugging data is thrown as an exception by the logger. Default is false. <br>***Note:*** Enabling this setting will result in dropped telemetry whenever an internal error occurs. This can be useful for quickly identifying issues with your configuration or usage of the SDK. If you do not want to lose telemetry while debugging, consider using `consoleLoggingLevel` or `telemetryLoggingLevel` instead of `enableDebug`. |
```
Comment (jpiyali) [inline AISKU/README.md]: should this be, sdk throws caught exceptions instead of just logging them?

File: AISKU/README.md (hunk 1 above)
Comment (markwolff) [reply AISKU/README.md]: The parameter fields are manually specified. It queries for the script and calcs a size and displays as svg. Actually I should change this to prod url

File: AISKU/README.md (hunk 2 above)
Comment (markwolff) [reply AISKU/README.md]: Agreed. Updating to clarify

**Pull request merged**